student-resumes/kian-resume/Resume.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html> 
+<html> 
+
+<head> 
+    <title>HTML Tutorial</title> 
+</head> 
+
+<body style="text-align: center"</body>
+
+    <h1>KIAN MURRAY</h1> 
+</body> 
+
+<body>
+    <font size="-4"> <h1>KianM@fake-email.com | (123) 456-7890 | References Available Upon Request</h1> 
+</body>
+
+<body>
+    <h1>___________________________________________________________________________________</h1> 
```

Comment:
You can accomplish this with a `<hr>` tag as well! Check it out here: https://www.w3schools.com/tags/tag_hr.asp

Reply:
lgtm though - nice work!